float32 x
float32 y
float32 size
float32 ttl
float32 dist_to
int32 id
int8 source_id